CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ All notable changes to this project will be documented in this file. Do not remo
   * [#622](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/622) docs: add missing fetch instrumentation ([@meteorlxy](https://github.com/meteorlxy))
   * [#553](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/553) feat: Add NestJS instrumentation ([@Rauno56](https://github.com/Rauno56))
   * [#588](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/588) chore: adding instrumentation for connect ([@obecny](https://github.com/obecny))
+  * [#1091](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1091) feat(express): add Express requestHook support ([@luismiramirez](https://github.com/luismiramirez))
```

Comment:
I think for contrib repo you don't need to update the CHANGELOG.md file. it is generated automatically upon publish.